native/src/hooks/useLoadCityContent.ts

```diff
@@ -27,6 +27,7 @@ import useSnackbar from './useSnackbar'
 type Params = {
   cityCode: string
   languageCode: string
+  refreshLocalNews?: boolean
```

Comment:
🔧  Can we call this something like `refreshForLocalNews`? `refreshLocalNews` sounds to me like it should be a function.